ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -120,3 +120,62 @@ SENSOR_FDPM:
     - local_air_temperature_sensor
     - battery_charge_percentage_sensor
     - signal_strength_percentage_sensor
+
+SENSOR_CO_CO2_ATM:
+  description: "Atmosphere CO and CO2 combined sensor with local temp and RH readings."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  uses:
+  - thirtysecondrolling_average_zone_air_co_concentration_sensor
+  - zone_air_co_concentration_sensor
+  - zone_air_co2_concentration_sensor
+  - local_air_relative_humidity_sensor
+  - local_air_temperature_sensor
+
+SENSOR_FORM_ATM:
+  description: "Atmosphere formaldehyde combined sensor with local temp and RH readings."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  uses:
+  - zone_air_formaldehyde_concentration_sensor
+  - local_air_relative_humidity_sensor
+  - local_air_temperature_sensor
+
+SENSOR_OZONE_NO_ATM:
+  description: "Atmosphere ozone and NO2 combined sensor with local temp and RH readings."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  uses:
+  - adjusted_zone_air_no2_concentration_sensor
+  - zone_air_ozone_concentration_sensor
+  - zone_air_voc_concentration_sensor
+  - zone_air_no2_concentration_sensor
+  - zone_air_no2_probability_percentage_sensor
+  - local_air_relative_humidity_sensor
+  - local_air_temperature_sensor
+
+SENSOR_PM_ATM:
+  description: "Atmosphere PM 0.5, PM 2.5, etc. combined sensor with local temp and RH readings."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  uses:
+  - thirtysecondrolling_average_particle_diameter_sensor
+  - zone_air_relative_humidity_sensor
```

Comment:
more thoughts about "zone":  the case for this prefix was pretty clear-cut when strictly talking about HVAC equipment, but when we're talking about a standalone sensor, the expectations for where it is placed become less clear.  I'm not sure we 100% solve this in the current PR, but perhaps we think ahead to what the ultimate solution is and see how it informs the current PR.  My first idea:
```
SPECIFIC_SENSOR:
  implements:
  - ABSTRACT_SENSOR:
    air_temperature_sensor:zone_air_temperature_sensor
```
The idea above being that you could remap fields to be more specific when inheriting a type, allowing us to not overspecify the sensor to start out with.

Reply:
I like this idea, but don't think we are going to solve it here. I will add a spike to think about this one.